.pre-commit-config.yaml

```diff
@@ -16,6 +16,13 @@ repos:
       - id: check-yaml
       - id: no-commit-to-branch
       - id: mixed-line-ending
+  - repo: local
+    hooks:
+      - id: sensitive-data-leak
+        name: Sensitive data leak hook
+        entry: (?<!www)\.sbb\.ch
```

Comment:
it should check for:
- all sbb.ch except www.sbb.ch and polarion-opensource@sbb.ch
- u- and e-numbers

Reply:
Updated.
Must be split into 2 checks due to "look-behind requires fixed-width pattern" error.